content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -110,86 +51,57 @@ follow
 etcd (the cluster database) and the API server (which the kubectl CLI
 communicates with).
 
-To initialize the master, first choose the pod network plugin you want and check if it requires any parameters to be passed to kubeadm while initializing the cluster. Pick one of the machines you previously installed
-kubeadm on, and run:
+1. Choose a pod network add-on, and verify whether it requires any arguments to 
+be passed to kubeadm initialization. Depending on which
+third-party provider you choose, you might need to set the `--pod-network-cidr` to
+a provider-specific value. See [Installing a pod network add-on](#pod-network).
+1. (Optional) Unless otherwise specified, kubeadm uses the network interface associated 
+with the default gateway to advertise the master's IP. To use a different 
+network interface, specify the `--apiserver-advertise-address=<ip-address>` argument 
+to `kubeadm init`. To deploy an IPv6 Kubernetes cluster using IPv6 addressing, you 
+must specify an IPv6 address, for example `--apiserver-advertise-address=fd00::101`
+
+Now run:
 
 ```bash
-kubeadm init
+kubeadm init <args> 
 ```
 
 **Notes:**
 
-- Please refer to the [kubeadm reference guide](/docs/reference/setup-tools/kubeadm/kubeadm/) if you want to
-read more about the flags `kubeadm init` provides. You can also specify a
-[configuration file](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file) instead of using flags.
-- You need to choose a Pod Network Plugin in the next step. Depending on what
-third-party provider you choose, you might have to set the `--pod-network-cidr` to
-something provider-specific. The tabs below will contain a notice about what flags
-on `kubeadm init` are required.
-- Unless otherwise specified, kubeadm uses the default gateway's network interface
-to advertise the master's IP. If you want to use a different network interface, specify
-`--apiserver-advertise-address=<ip-address>` argument to `kubeadm init`. To deploy an IPv6 Kubernetes cluster using IPv6 addressing, you must specify an IPv6, e.g. `--apiserver-advertise-address=fd00::101`
-- If you would like to customise control plane components including optional IPv6 assignment to liveness probe for control plane components and etcd server, you can do so by providing extra args to each one, as documented [here](/docs/admin/kubeadm#custom-args).
-- `kubeadm init` will first run a series of prechecks to ensure that the machine
-is ready to run Kubernetes. It will expose warnings and exit on errors. It
-will then download and install the cluster database and control plane
-components. This may take several minutes.
-- You can't run `kubeadm init` twice without tearing down the cluster in between
-([unless you're upgrading from v1.6 to v1.7](/docs/tasks/administer-cluster/kubeadm-upgrade-1-7/)),
-see [Tear Down](#tear-down).
+- For more information about `kubeadm init` arguments, see [the kubeadm reference guide.](/docs/reference/setup-tools/kubeadm/kubeadm/) 
```

Comment:
It seems weird for this to be a separate bullet. If I already ran `kubeadm init` then I've missed my chance to add extra arguments, right? Maybe this should be after the second bullet above about the optional network interface arguments.